### PR TITLE
[zh-cn]: update the tranlation of promise constructor

### PR DESCRIPTION
**Pull request by fuchunhui**

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/promise/promise/index.md [hunk 1]
```diff
@@ -99,9 +101,11 @@ readFilePromise("./data.txt")
   .catch((error) => console.error("读取文件失败"));
 ```
 
-### Resolver 函数
+`resolve` 和 `reject` 回调仅在 `executor` 函数的范围内可用，这意味着在构造 promise 之后无法访问它们。如果你想在决定如何解决之前构造 promise，可以使用 {{jsxref("Promise.withResolvers()")}} 方法，该方法公开了 `resolve` and `reject` 函数。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/promise/promise/index.md:104]:
```suggestion
`resolve` 和 `reject` 回调仅在 `executor` 函数的作用域内可用，这意味着在构造 promise 之后无法访问它们。如果你想在决定如何解决之前先构造 promise，可以使用 {{jsxref("Promise.withResolvers()")}} 方法，该方法暴露了 `resolve` and `reject` 函数。
```

File: files/zh-cn/web/javascript/reference/global_objects/promise/promise/index.md [hunk 2]
```diff
@@ -132,7 +136,7 @@ new Promise((resolve, reject) => {
 
 但是在 `resolve(thenable)` 的情况中，有如下区别：
 
-1. `resolve` 函数是同步调用的，因此再次调用 `resolve` 或 `reject` 函数没有任何影响，即使通过 `anotherPromise.then()` 绑定的处理程序尚未被调用。
+1. `resolve` 函数是同步调用的，因此再次调用 `resolve` 或 `reject` 函数没有任何影响，即使通过 `anotherPromise.then()` 绑定的处理程序尚未被调用也没影响。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/promise/promise/index.md:139]: Already in "...因此再次调用 `resolve` 或 `reject` 函数没有任何影响"
```suggestion
1. `resolve` 函数是同步调用的，因此再次调用 `resolve` 或 `reject` 函数没有任何影响，即使通过 `anotherPromise.then()` 绑定的处理程序尚未被调用。
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions.

File: files/zh-cn/web/javascript/reference/global_objects/promise/promise/index.md (hunk 1 above)
Comment (fuchunhui) [reply files/zh-cn/web/javascript/reference/global_objects/promise/promise/index.md:104]: cool，got it.

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Thank you for your contributions! @fuchunhui

**Pull request merged**